package.json

```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.4.13",
-    "@craco/craco": "^6.4.3",
+    "@craco/craco": "7.0.0-alpha.3",
```

Comment:
Why this bump?

Reply:
I'm trying to resolve it a build issue https://github.com/FuelLabs/block-explorer-v2/runs/5788640372?check_suite_focus=true

It doesn't make sense.  I know CRACO doesn't support `react scripts v5`, but why did the CI start breaking?

Reply:
I made some tests with the branch master and this is due to an incorrect update `package-lock.json` it was working. I had to remove the `package-lock` on the master locally and try to install it and got the same error.

Nice...

Reply:
I tried to downgrade react scripts, but it's a bit of a mess because of the TS version.  Can we keep this version of craco?